docker/docker-compose.yml

```diff
@@ -1,34 +1,72 @@
 version: '3'
 services:
+  spark-submit:
+    image: metorikku/metorikku
+    container_name: spark-submit
+    ports:
+      - "4040:4040"
+    environment:
+      - SPARK_MASTER_HOST=spark-master
+      - SPARK_MASTER_PORT=7077
+      - SPARK_WEBUI_PORT=8080
+      - SPARK_UI_PORT=4040
+      - NUM_WORKERS=2
+      - JSON_LOG=true
+      - GRAPHITE_HOST=graphite.local
+      - GRAPHITE_PORT=3001
+      - SUBMIT_COMMAND=spark-submit --class com.yotpo.metorikku.Metorikku metorikku.jar -c examples/movies.yaml
+    entrypoint:
+      - /entrypoint-submit.sh
   spark-master:
     image: metorikku/metorikku
     container_name: spark-master
+    depends_on:
+      - spark-submit
```

Comment:
I think it's other way around, move this dependency to submit, submit depends on master, worker depends on master as well (but not really so you can actually  remove this)

Reply:
Or is it actually causing it all to die after submit dies?

Reply:
when using depends_on the docker-compose command starts the services in the dependency order.
and if you run docker-compose up master, it will also start the submit...